packages/runtime/container-runtime/src/pendingStateManager.ts

```diff
@@ -374,12 +373,12 @@ export class PendingStateManager implements IDisposable {
 				asEmptyBatchLocalOpMetadata(localOpMetadata)?.emptyBatch === true,
 				"Expected empty batch marker",
 			);
+			return [];
 		}
 
-		// Note this will correctly return empty array for an empty batch
 		return batch.messages.map((message) => ({
 			message,
-			localOpMetadata: this.processNextPendingMessage(message.sequenceNumber, message),
+			localOpMetadata: this.processNextPendingMessage(message.sequenceNumber),
```

Comment:
Was this intentionally removed? Looks like an accident because this method arg is now never used.

Reply:
Hmm good catch!